RecoTracker/MkFit/plugins/MkFitGeometryESProducer.cc

```diff
@@ -0,0 +1,69 @@
+#include "FWCore/Framework/interface/ModuleFactory.h"
+#include "FWCore/Framework/interface/ESProducer.h"
+
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
+#include "RecoTracker/TkDetLayers/interface/GeometricSearchTracker.h"
+#include "RecoTracker/Record/interface/TrackerRecoGeometryRecord.h"
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+
+#include "RecoTracker/MkFit/interface/MkFitGeometry.h"
+
+// mkFit includes
+#include "ConfigWrapper.h"
+// TODO: the need for this define MUST be removed
+// e.g. make it a build configure time choice instead of build time shoice
+#define USE_MATRIPLEX
+// TODO: this include causes "warning: ignoring #pragma omp simd" from Matriplex.h, can we try to hide the Matriplex from CMSSW?
+#include "mkFit/SteeringParams.h"
```

Comment:
These two TODOs are actually something that would be good to address before the PR to CMSSW. Maybe it would be straightforward to extract the `IterationsInfo` and whatever it needs out from `StreeringParams.h`? Or do `IterationsInfo` have some dependence to classes defined in `Matrix.h`?

@osschar

Reply:
I will fix those today.

Reply:
This has been fixed in #314, I moved FindingFoos into separate header and moved classes with Iteration in their names to IterationConfig.h.

Once merged, you can remove the #define and include mkFit/IterationConfig.h instead of SteeringParams.h.